.travis.yml

```diff
@@ -22,7 +22,8 @@ before_install:
   - wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar --output-document=php-cs-fixer.phar
 
 install:
-  - composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
+  - composer global show hirak/prestissimo -q || travis_retry composer global require hirak/prestissimo --prefer-dist --no-interaction --no-progress $COMPOSER_FLAGS
```

Comment:
ref https://github.com/sonata-project/dev-kit/issues/173#issuecomment-377817983

if `prestissimo` is already installed, we don't need to install it again.
so same time as project dependencies are cached and `prestissimo` is not bringing that big benefits, we could cache it as well, so we won't lose any time if it's already there.

so with this change, we have 2 scenarios:
- no composer cache, install `prestissimo` plugin and then deps using plugin
- composer cache is there, don't install plugin and install deps from cache

Reply:
no need for composer flags here AFAIK, only for the composer update command

Reply:
sure, I didn't found where that env var was set and what you provide as a value, so I kept it as is. dropping...